File: ejercicios/python/ejercicio060.py
def sumador_iterativo(numero):
   suma = 0
   for i in range(0,numero+1):
      suma += i
   return suma

if __name__ == "__main__":
   numero = int(input("ingrese el numero : "))
   print ("la suma de los primeros " + str(numero) + " numeros enteros es : " + str(sumador_iterativo(numero)))

File: ejercicios/c++/ejercicio060.cpp
#include <iostream>
using namespace std;

long sumador_iterativo(int numero){
	long suma = 0;
	for(int i = 0; i <= numero; i++){
		suma += i;
	}
	return suma;
}

int main() {
	int numero;
	cout << "ingrese el numero : ";
	cin >> numero;
	
	cout << "la suma de los " << numero << " primeros numeros naturales es : " << sumador_iterativo(numero) << endl;
	
	return 0;
}



